src/blockchain/payments.js

```diff
@@ -112,6 +118,7 @@ const payments = app => ({
       recipientAddress: recipient,
       milestoneId,
       campaignId,
+      donationId,
```

Comment:
Every `PaymentAuthorized` event can affect multiple donations and 
```
      donationModel.findOne({ txHash: reference }, ['ownerTypeId']),
``` 
can return different donation on every run. So saving donationId would not be a solution

Reply:
You are right, so there is some complexities, we should talk about the logic when you're back